src/constants/grid.js

```diff
@@ -18,8 +18,27 @@ const getGridExtent = () => {
 
 const EARTH_CIRCUIT = 40075000 //in meters
 
+//Tile size in degrees
+const LEVEL_0_TILE_SIZE = BASE_SIZE / 2;
+
+//size of tile in pixels
+const PIXEL_TILE_SIZE = 250;
+
+//resolutions
+//Resolution is number of meters displayed inone pixel.
+
+// Maximum resolution for level 0. Level 0 has two tiles of size 180° x 180°
+const maxResolution = EARTH_CIRCUIT / PIXEL_TILE_SIZE;
```

Comment:
Doesn't Earth's circumference correspond to 360°?

Reply:
Fixed comment

Reply:
I'm still not sure about this. Earth's circumference spans two tiles in level 0, and you're dividing it by (one) tile size. Is that how it's supposed to be?

Reply:
Yes, it should be half of the circumference, otherwise it could be misleading.